app/screens/post_options/reaction_bar/pick_reaction/index.tsx

```diff
@@ -32,19 +38,31 @@ type PickReactionProps = {
 const PickReaction = ({openEmojiPicker, width, height}: PickReactionProps) => {
     const theme = useTheme();
     const styles = getStyleSheet(theme);
+
+    const highlightedStyle = useCallback(({pressed}: PressableStateCallbackType) => pressed && styles.highlight, [styles.highlight]);
+    const pickReactionStyle = useMemo(() => [
+        styles.container,
+        {
+            width,
+            height,
+        },
+    ], [width, height]);
+
     return (
-        <View
-            style={[styles.container, {
-                width, height,
-            }]}
-            testID='post_options.reaction_bar.pick_reaction.button'
+        <Pressable
```

Comment:
I attempted to use Pressable here since it was used for the other quick reactions....not sure if I did this right though

Reply:
If the behaviour is the expected and it is used in other similar places, I see no reason not to use it.